<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkShiftCreatorWidget</class>
 <widget class="QWidget" name="WorkShiftCreatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkShiftCreatorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="p_buttons_layout">
     <item>
      <widget class="QPushButton" name="p_button_CreateNew">
       <property name="text">
        <string>Create New...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_Open">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_Save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_SaveAs">
       <property name="text">
        <string>Save as...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="p_group_Shifts">
     <property name="title">
      <string>Shifts</string>
     </property>
     <layout class="QVBoxLayout" name="p_layout_Shifts">
      <item>
       <widget class="QListWidget" name="p_list_Shifts"/>
      </item>
      <item>
       <widget class="QPushButton" name="p_button_AddShift">
        <property name="text">
         <string>Add shift</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p_button_DeleteShift">
        <property name="text">
         <string>Delete shift</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="p_group_ShiftsData">
     <property name="title">
      <string>Work shift settings</string>
     </property>
     <layout class="QGridLayout" name="p_grid_Form">
      <item row="2" column="1">
       <widget class="QComboBox" name="p_comboBox_Position"/>
      </item>
      <item row="4" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndDate"/>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="p_comboBox_ID_Workers"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="p_lbl_Position">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="p_lbl_StartDate">
        <property name="text">
         <string>Start date</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartDate"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="p_line_ID_Shift"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="p_lbl_EndDate">
        <property name="text">
         <string>End date</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_ID_Shift">
        <property name="text">
         <string>Shift ID</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="p_lbl_ID_Worker">
        <property name="text">
         <string>Workers ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
